lib/tasks/build.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'build_utils'
+include BuildUtils
+
+namespace :build do
+
+  desc "Create buildbox for Open OnDemand"
+  task :build_box, [:platform, :version] do |task, args|
+    platform = "#{args[:platform] || "ubuntu"}"
+    image_tag = "#{platform}-#{args[:version] || '20.04-1'}"
```

Comment:
Here's the initial notes on the buildbox.  

buildboxes are `ood-buildbox:#{platform}-#{version}`. Version is their version + ours (which I figure we'll increment). 20.04 being ubuntu focal and `-1` is just our first version of it.

I'm happy to keep this here now, but some things we'll have to figure out
* how to tag & push these images
* How to update them. In this case how would we start building in `20.04-1`, how do we start using `20.04-2`.